presto-hive/pom.xml

```diff
@@ -258,6 +258,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
+            <version>2.14.3</version>
```

Comment:
I see this is already included in root pom - https://github.com/prestodb/presto/pull/23753/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R795

Do we still need version here? Is it coming in higher version as transitive dependency from other place?

Reply:
The specific issue with jackson-databind 2.15.0 and above affects Presto-Hive tests due to deserialization changes. Downgrading jackson-databind to 2.14.3 here has stabilized the tests without impacting the rest of the engine.

Reply:
Seems the serialized test data should be updated instead. This is going in the wrong direction